src/bindings/python/src/compatibility/openvino/__init__.py

```diff
@@ -2,4 +2,71 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
+    # Internal modules
+    from openvino import test_utils as test_utils
+except ImportError:
+    pass
+
+try:
+    # Import all public modules
+    from openvino import runtime as runtime
+    from openvino import frontend as frontend
+    from openvino import helpers as helpers
+    from openvino import preprocess as preprocess
+    from openvino import utils as utils
+    from openvino.runtime import properties as properties
+    # Import old API
+    # TODO: remove in 2024.0
+    from openvino import inference_engine as inference_engine
+
+    # Required for Windows OS platforms
+    from openvino.utils import _add_openvino_libs_to_search_path
+
+    _add_openvino_libs_to_search_path()
```

Comment:
should this statement be even before `from openvino import test_utils as test_utils ` ?
E.g. we need to set libraries path first, then try to import functionality. 

I suppose if `test_utils` depends on some DLLs, it can fail without `_add_openvino_libs_to_search_path `, can not it?

Reply:
Good point.